lib/building_oregon/file_distributor.rb

```diff
@@ -0,0 +1,32 @@
+module BuildingOregon
+  class FileDistributor
+    attr_accessor :base_path, :identifier, :depth, :extension
+
+
+    def initialize(identifier)
+      @identifier = identifier.to_s
+      @depth = 2
+      @source_site = "http://oregondigital.library.oregonstate.edu/"
```

Comment:
Instead of defining these statically, I'd use the accessors when you use it to set the appropriate properties.